modules/mod_ginger_base/controllers/controller_rest_edges.erl

```diff
@@ -24,14 +25,28 @@
 %%% Resource functions
 %%%-----------------------------------------------------------------------------
 
-init([Args]) ->
-    Mode =  maps:get(mode, Args),
+init(Args) ->
+    Mode = proplists:get_value(mode, Args),
     {ok, #state{mode = Mode}}.
 
 service_available(Req, State) ->
     Context = z_context:continue_session(z_context:new(Req, ?MODULE)),
     {true, Req, State#state{context = Context}}.
 
+is_authorized(Req, State) ->
+    case lists:member(wrq:method(Req), ['POST', 'DELETE']) of
+        false ->
+            {true, Req, State};
+        true ->
+            Context = State#state.context,
+            case z_acl:user(Context) of
+                undefined ->
+                    {{halt, 401}, Req, Context};
+                _Id ->
+                    {true, Req, State}
```

Comment:
Instead of just true, we'd probably want to have an authorization function taking the user id here.

Reply:
To further check if the action is allowed?

Reply:
Yes, either by calling into the ginger acl rules or by calling a function that implements a rest specific rulebase (e.g. "can only change own data" or something).

Reply:
Ah.. the zotonic functions for manipulating edges already have acl checks, so indeed your suggestion of adding tests while leaving this `true` is a good idea.